app/invocation/invocation_action_card.tsx

```diff
@@ -100,6 +99,13 @@ export default class ActionCardComponent extends React.Component<Props, State> {
     );
   }
 
+  getCacheAddress() {
+    if (this.props.model.optionsMap.get("remote_cache")) {
+      return this.props.model.optionsMap.get("remote_cache").replace("grpc://", "");
```

Comment:
In prod, it'll be `grpcs://` instead of `grpc://` -- can you also find & replace `grpcs://` prefixes here and below?